.github/workflows/tag-new-release.yml

```
@@ -0,0 +1,65 @@
+name: Release a new version
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release Version (e.g., 1.2.3)'
+        required: true
+        type: string
+      next_version:
+        description: 'Next Version (e.g., 1.2.4 or skip)'
+        required: true
+        type: string
+
+env:
+  LC_ALL: "en_US.UTF-8"
+
+jobs:
+  release-and-tag:
+    if: github.repository == 'line/centraldogma-python'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Validate inputs
+        run: |
+          if [[ ! "${{ inputs.release_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Error: 'release_version' should be in SemVer format."
+            exit 1
+          fi
+          if [[ ! "${{ inputs.next_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Error: 'next_version' should be in SemVer format."
```

Even if `skip` is allowed by `inputs.next_version` description and following implementation, this validation always fail if `skip` is given.
```suggestion
          if [[ ! "${{ inputs.next_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ && "${{ inputs.next_version }}" != "skip" ]]; then
            echo "Error: 'next_version' should be in SemVer format or 'skip'."
```

---

Yeah, I need to fix that. 😉